src/contextMenu.component.ts

```diff
@@ -23,10 +23,10 @@ export interface ILinkConfig {
   html: (item: any) => string;
 }
 export interface MouseLocation {
-  left?: string;
-  marginLeft?: string;
-  marginTop?: string;
-  top?: string;
+  left?: number;
+  marginLeft?: number;
+  marginTop?: number;
+  top?: number;
 }
```

Comment:
Sure, no problem - critical inspection is appreciated :-)

The reason is that I do some calculations on these values in `clickedOutside`. If they were strings, I'd have to convert them to numbers. The way I see it, they only need to be strings in `locationCss`, which is where the conversion happens now.